index.html

```diff
@@ -797,7 +800,7 @@ <h5 class="card-title">
           <header class="section-header" style="margin-bottom: 35px">
             <div>
               <h2 class="primary-color">Our Sponsors</h2>
-              <p class="lead">
+              <p class="lead"></p>
```

Comment:
this closing tag here is unnecessary. Could you remove it so I can approve this PR?

Reply:
Made the changes